src/engine/MaxConstraint.cpp

```diff
@@ -34,6 +34,7 @@
 MaxConstraint::MaxConstraint( unsigned f, const Set<unsigned> &elements )
     : _f( f )
     , _elements( elements )
+    , _initialElements ( elements )
```

Comment:
Cosmetics: please remove the space before the '('

Reply:
Yes. I did.